sharpy/utils/plotutils.py

```diff
@@ -1,6 +1,8 @@
 """Plotting utilities
 """
 import numpy as np
+from mpl_toolkits.mplot3d import axes3d
+import matplotlib.pyplot as plt
```

Comment:
Since we do not include these in the environment file (and I believe it used to cause issues with Docker), could we include these in a `try: ... except ModuleNotFoundError:` clause. Could you put these under the `if not plotly`?

Reply:
Done, thanks